### PR TITLE
Update iojs to v1.1.0

### PR DESCRIPTION
**Pull request by pesho**

### CODE REVIEW

File: library/iojs [hunk 1]
```diff
@@ -2,15 +2,24 @@
 
 1.0.4: git://github.com/iojs/docker-iojs@39ab80a90ff9addd935d3acf1f5bc3eff49d6ffc 1.0
```
Comment (tianon) [inline library/iojs]: Is there a reason to keep `1.0`/`1.0.4` here?  For example, if a problem with `1.0.4` is discovered, will there be a `1.0.5`?

We won't delete the tags from the Hub unless it's explicitly requested by the maintainer that we do so, so people will still be able to use `iojs:1.0.4`, it just won't be listed under "Supported tags" in the description, and won't be rebuilt.

Comment (yosifkit) [inline library/iojs]: @tianon, see discussion over here: https://github.com/iojs/docker-iojs/pull/18

Comment (pesho) [inline library/iojs]: @tianon no, according to https://github.com/iojs/docker-iojs/pull/18#issuecomment-72789460 we will remove 1.0.x. Do you want to delay this PR for that?

Comment (tianon) [inline library/iojs]: Indeed, in https://github.com/iojs/docker-iojs/pull/18#issuecomment-72789460 it sounds to me like it's 100% certain there will not be a 1.0.5. :+1:

Comment (tianon) [inline library/iojs]: I would like the "Supported tags" section to be accurate.  If you promise a
PR for that will come soon, I'm OK with this one as-is for now. :+1:

Comment (pesho) [inline library/iojs]: Ok, i'll prepare a PR for that shortly.

**Comment by tianon** [comment]
LGTM

Build test of #455; ea0d5a7bea31f86500d694913dbaf37683ee4e39 (`iojs`):

```console
$ bashbrew build https://raw.githubusercontent.com/docker-library/official-images/ea0d5a7bea31f86500d694913dbaf37683ee4e39/library/iojs
Fetching iojs (git://github.com/iojs/docker-iojs) ...
Processing iojs:1.0.4 ...
Processing iojs:1.0 ...
Processing iojs:1.0.4-onbuild ...
Processing iojs:1.0-onbuild ...
Processing iojs:1.0.4-slim ...
Processing iojs:1.0-slim ...
Processing iojs:1.1.0 ...
Processing iojs:1.1 ...
Processing iojs:1 ...
Processing iojs:latest ...
Processing iojs:1.1.0-onbuild ...
Processing iojs:1.1-onbuild ...
Processing iojs:1-onbuild ...
Processing iojs:onbuild ...
Processing iojs:1.1.0-slim ...
Processing iojs:1.1-slim ...
Processing iojs:1-slim ...
Processing iojs:slim ...
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**